src/index.html

```diff
@@ -3,12 +3,98 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <div class="menu__content">
+        <a href="#home">
+          <img
+            class="MOYO__logo"
+            src="./images/logo.png"
+            alt="MOYO logo"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__box">
+              <a
+                href="#apple"
+                class="nav__link is-active">
```

Comment:
please write each attribute from new line **only if** element has long attribute values or number of attributes is more than 2 (checklist - point 13)

Reply:
fix the whole HTML file accordingly